clients/da-vinci-client/src/main/java/com/linkedin/davinci/client/DaVinciRecordTransformer.java

```diff
@@ -206,6 +205,33 @@ public final boolean getStoreRecordsInDaVinci() {
     return storeRecordsInDaVinci;
   }
 
+  /**
+   * Returns the schema for the key used in {@link DaVinciClient}'s operations.
+   *
+   * @return a {@link Schema} corresponding to the type of {@link K}.
+   */
+  public final Schema getKeySchema() {
+    return keySchema;
+  }
+
+  /**
+   * Returns the schema for the output value used in {@link DaVinciClient}'s operations.
+   *
+   * @return a {@link Schema} corresponding to the type of {@link V}.
+   */
+  public final Schema getOriginalValueSchema() {
```

Comment:
How about `getInputValueSchema`, or `getLatestInputValueSchema`, to contrast with `getOutputValueSchema`? "Original" can be confusing IMHO.

Let's also fix the first line of the JavaDoc where it says "output".